python/lsst/daf/butler/cli/cliLog.py

```diff
@@ -182,7 +182,12 @@ def initLog(
         else:
             logging.basicConfig(level=logging.WARNING, format=cls.pylog_normalFmt, style="{")
 
-        # Initialize root logger level.
+        # Initialize the root logger. Calling this ensures that both
+        # python loggers and lsst loggers are consistent in their default
+        # logging level.
+        cls._setLogLevel(".", "WARNING")
```

Comment:
I'm wondering a bit how this interacts with the `basicConfig(level=` above, but it looks OK.